### ISSUE TITLE
Quasi-XML parser prevents proper query of unicode object names

### ISSUE THREAD

**Issue by d-w-moore** [irods/python-irodsclient#318]
When using Python 2.7 on  iRODS 4.2.11 ...
`ucreate.py` is:
```python
from irods.test.helpers import make_session
from irods.message import (ET, XML_Parser_Type)
with make_session() as ses:
    ET(XML_Parser_Type.QUASI_XML, ses.server_version)
    dataname = u"r{eacute}prouv{eacute}".format(eacute = u'\u00e9')
    homepath="/{0.zone}/home/{0.username}".format(ses)
    homecoll=ses.collections.get(homepath)
    d = ses.data_objects.create(u"{homepath}/{dataname}".format(**locals()))
    print("THIS WON'T PRINT: object has id {0.id}".format(d))
```
This script creates the data object but on query to *retrieve* the object reference, things blow up:
```
(py2) linux:~$ python ucreate.py
Traceback (most recent call last):
  File "thing.py", line 10, in <module>
    ses.data_objects.create(u"{homepath}/{dataname}".format(**locals()))
  File "/home/daniel/python-irodsclient/irods/manager/data_object_manager.py", line 245, in create
    return self.get(path)
  File "/home/daniel/python-irodsclient/irods/manager/data_object_manager.py", line 116, in get
    results = query.all() # get up to max_rows replicas
  File "/home/daniel/python-irodsclient/irods/query.py", line 189, in all
    result_set = self.execute()
  File "/home/daniel/python-irodsclient/irods/query.py", line 175, in execute
    results = result_message.get_main_message(GenQueryResponse)
  File "/home/daniel/python-irodsclient/irods/message/__init__.py", line 314, in get_main_message
    msg.unpack(ET().fromstring(self.msg))
  File "/home/daniel/python-irodsclient/irods/message/quasixml.py", line 149, in fromstring
    s = s.encode('utf-8')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1757: ordinal not in range(128)
(py2) linux:~$ 
```

**Comment by d-w-moore** [comment]
This also fails. (No put, just a query) when the unicode-named object exists.
```
from irods.test.helpers import make_session
from irods.message import (ET, XML_Parser_Type)

with make_session() as ses:
    ET(XML_Parser_Type.QUASI_XML, ses.server_version)
    home_path = '/{0.zone}/home/{0.username}'.format(ses)
    home = ses.collections.get( home_path )
    l = list(home.walk())
```

**Comment by d-w-moore** [comment]
Ok That's all fixed with  #319
